ranger/core/actions.py

```diff
@@ -1591,14 +1592,29 @@ def _recurse_hardlinked_tree(self, source_path, target_path):
                 link(source_path,
                      next_available_filename(target_path))
 
-    def paste(self, overwrite=False, append=False, dest=None):
+    def paste(self, conflict='rename_ext', append=False, dest=None, overwrite=False):
```

Comment:
Don't change the argument order. There's no good reason to.

Reply:
I also don't like the string argument, too error prone. Let's just pass a function.